scorecard.test.js

```diff
@@ -0,0 +1,22 @@
+const Scorecard = require('./scorecard');
+
+describe('.pins', () => {
+  it('adds pins', () => {
```

Comment:
I'm not sure this is quite right here? .pins method doesn't add pins, I think that's the recordPins method? Your pins method is for displaying the scorecard, so maybe you could break this into 2 tests to test each method individually.